base/pcp/Dockerfile

```diff
@@ -2,9 +2,11 @@ FROM registry.access.redhat.com/rhel7
 ENV LANG=en_US.utf8
 
 COPY pcp.repo /etc/yum.repos.d/pcp.repo
-RUN yum install -y pcp pcp-pmda-prometheus pcp-manager pcp-debuginfo \
+RUN yum-config-manager --enable rhel-7-server-extras-rpms && \
+    yum install -y pcp pcp-pmda-prometheus pcp-manager pcp-debuginfo \
                    pcp-export-pcp2zabbix pcp-system-tools pcp-webapi \
-                   pcp-webapps pcp-pmda-postgresql && yum clean all && \
+                   pcp-webapps pcp-pmda-postgresql kubernetes-client && \
```

Comment:
Try a `origin-clients` install instead of kubernetes-client here?

Reply:
origin-clients is not available in RHEL